gax-java/gax/src/main/java/com/google/api/gax/tracing/BaseApiTracer.java

```diff
@@ -33,7 +33,10 @@
 import org.threeten.bp.Duration;
 
 /**
- * A base implementation of {@link ApiTracer} that does nothing.
+ * A base implementation of {@link ApiTracer} that does nothing. With the deprecation of Java 7
+ * support, all the methods in {@link ApiTracer} are now made default, we no longer need a base
+ * class that does nothing. This class should be removed once all the references to it are removed
+ * in Google Cloud Client Libraries.
```

Comment:
Perhaps we can create a ticket for this so we don't forget about it. @ddixit14 

If I'm understanding this correctly, once stage 1 is complete, we can remove this class.

Reply:
sure, I'll create a ticket for this.

Reply:
I'm planning to deprecate `BaseApiTracer` in a follow up PR, it is an `InternalApi` and only used by Bigtable outside  of our team. So we can could deprecate it anytime want, and remove it after Bigtable removes the usage of it.